postgres/client.go

```diff
@@ -0,0 +1,13 @@
+package postgres
+
+import (
+	"database/sql"
+	_ "github.com/go-sql-driver/mysql"
+	_ "github.com/godror/godror"
+	_ "github.com/lib/pq"
+	"net/url"
+)
+
+func SetConf(u *url.URL) (*sql.DB, error) {
+	return sql.Open(u.Scheme, u.String())
```

Comment:
Not all schemas match go drivers, we should use switch to adjust driver to schema. For example:

`oracle` -> `godor`